fast_api/routes/organization.py

```diff
@@ -120,8 +124,11 @@ def all_active_admin_messages(request: Request, limit: int = 100) -> str:
 
 @router.get("/all-admin-messages")
 def all_admin_messages(request: Request, limit: int = 100) -> str:
+
```

Comment:
maybe we should also restrict this endpoint for admins, not sure about the purpose